src/OrchardCore/OrchardCore.Localization.Core/ModularPoFileLocationProvider.cs

```diff
@@ -47,27 +49,67 @@ public IEnumerable<IFileInfo> GetLocations(string cultureName)
             foreach (var extension in extensions)
             {
                 // From [Extension]/Localization
-                yield return _fileProvider.GetFileInfo(PathExtensions.Combine(extension.SubPath, _resourcesContainer, poFileName));
+                yield return _fileProvider.GetFileInfo(Combine(extension.SubPath, _resourcesContainer, poFileName));
             }
 
             // Load global .po files from /Localization
-            yield return _fileProvider.GetFileInfo(PathExtensions.Combine(_resourcesContainer, poFileName));
+            yield return _fileProvider.GetFileInfo(Combine(_resourcesContainer, poFileName));
 
             // Load tenant-specific .po file from /App_Data/Sites/[Tenant]/Localization
-            yield return new PhysicalFileInfo(new FileInfo(PathExtensions.Combine(_shellDataContainer, _resourcesContainer, poFileName)));
+            yield return new PhysicalFileInfo(new FileInfo(Combine(_shellDataContainer, _resourcesContainer, poFileName)));
 
             // Load each modules .po file for extending localization when using Orchard Core as a Nuget package
             foreach (var extension in extensions)
             {
                 // \src\OrchardCore.Cms.Web\Localization\OrchardCore.Cms.Web\fr-CA.po
-                yield return _fileProvider.GetFileInfo(PathExtensions.Combine(_resourcesContainer, extension.Id, poFileName));
+                yield return _fileProvider.GetFileInfo(Combine(_resourcesContainer, extension.Id, poFileName));
 
                 // \src\OrchardCore.Cms.Web\Localization\OrchardCore.Cms.Web-fr-CA.po
-                yield return _fileProvider.GetFileInfo(PathExtensions.Combine(_resourcesContainer, extension.Id + CultureDelimiter + poFileName));
+                yield return _fileProvider.GetFileInfo(Combine(_resourcesContainer, extension.Id + CultureDelimiter + poFileName));
 
                 // \src\OrchardCore.Cms.Web\Localization\fr-CA\OrchardCore.Cms.Web.po
-                yield return _fileProvider.GetFileInfo(PathExtensions.Combine(_resourcesContainer, cultureName, extension.Id + PoFileExtension));
+                yield return _fileProvider.GetFileInfo(Combine(_resourcesContainer, cultureName, extension.Id + PoFileExtension));
+            }
+        }
+
+        private static string Combine(string path1, string path2)
```

Comment:
Why not copying this file instead? https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Files/PathExtensions.cs

Reply:
I copy two functions from it, I don't think that I need the rest of them

Reply:
Remove the functions you don't require. I don't mind, it's just cleaner to identify that these are Extensions methods which should not necessarly be included inside a class. But either way, your call.

Reply:
Ok

Reply:
You can even "link" the file in the project. This way we'll use the same source code, always.

Reply:
Ya ya .. that will be better

Reply:
This will add
```
<ItemGroup>
    <Compile Include="..\OrchardCore.Infrastructure.Abstractions\Files\PathExtensions.cs" Link="PathExtensions.cs" />
  </ItemGroup>
```

Seb if it's fine for you I will push my changes, coz you had some doubt with `OrchardCore.Infrastructure.Abstractions` above